plugins/inputs/http_listener_v2/README.md

```diff
@@ -36,8 +36,19 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 ```toml @sample.conf
 # Generic HTTP write listener
 [[inputs.http_listener_v2]]
-  ## Address and port to host HTTP listener on
+  ## Address to host HTTP listener on
+  # can be prefixed by protocol tcp, or unix if not provided defaults to tcp
+  # if unix network type provided it should be followed by absolute path for unix socket
   service_address = ":8080"
+  # service_address = "tcp://:8443"
+  # service_address = "unix:///tmp/telegraf.sock"
```

Comment:
By convention we all 'single-hashed' comments carry "default values" so please really use double hashes for comments...
```suggestion
  ## Address to host HTTP listener on, optionally including the protocol (tcp or unix).
  ## "unix" addresses should contain the absolute path for the socket, e.g.
  ##   service_address = "unix:///tmp/telegraf.sock"
  service_address = "tcp://:8080"
```

Reply:
thanks will follow the convention from now on - [a239d9e](https://github.com/influxdata/telegraf/pull/15764/commits/a239d9ed32e4ab2e9c55f7fa5e511d1e71e014db)